bootloader/wscript

```diff
@@ -749,8 +749,8 @@ def build(ctx):
                                 ctx.env.PYI_SYSTEM + "-" + ctx.env.PYI_ARCH)
     install_path = os.path.normpath(install_path)
 
-    if machine() and ctx.env.DEST_OS != 'darwin':
-        install_path += '-' + machine()
+    if machine(ctx) and ctx.env.DEST_OS != 'darwin':
```

Comment:
Isn't `machine(ctx)`now always truthy?

Reply:
Ah, nevermind. It's truthy only if OS is linux.

Reply:
It returns `None` on non-Linux so no.